package.json

```diff
@@ -12,7 +12,7 @@
     "router",
     "event router"
   ],
-  "main": "dist/esp.js",
+  "main": "index.js",
```

Comment:
when you do a require('esp-js') what ever owns the require statement will look into packages.json to get the main file of your distribution. If that main file points to index.js, then that is an ES6 file and it will import all the ES6 resources. 

However if main points to dist/esp.js, then that's transpiled at build time and pushed to npm so consumers of the distribution don't need to worry about ES6